src/loaders/ObjectLoader.js

```diff
@@ -327,6 +327,19 @@ Object.assign( ObjectLoader.prototype, {
 
 						break;
 
+					case 'TubeGeometry':
+					case 'TubeBufferGeometry':
+
+						geometry = new Geometries[ data.type ](
+							new Curves[ data.path.type ]().fromJSON( data.path ),
```

Comment:
What if an instance of `CurvePath` was originally passed into `TubeBufferGeometry`?

Reply:
I hadn't noticed that `ShapeBufferGeometry` and `ExtrudeBufferGeometry` overwrote the toJSON method. I'll do that instead for TubeBufferGeometry.